app/Services/user/BlogService.php

```diff
@@ -42,8 +45,34 @@ public function show(int $id): Collection
         }
     }
 
+    public function update(object $request, object $blog): bool
```

Comment:
sao lại toàn object nhỉ?

Reply:
cái này bên controller cả $request và $blog em kiểm tra đều là object nên khi chuyền vào nó có kiểu dữ liệu là object ạ